### PR TITLE
Add additional schedulers to skip_registration

### PR DESCRIPTION
**Pull request by b10n1k**
Add schedulers for:
s390x
svirt-hyperv
svirt-hyperv-uefi
svirt-xen-hvm
svirt-xen-pv
uefi

- Related ticket: https://progress.opensuse.org/issues/58691
- Needles: 
- Verification run: 

[skip_registration](https://openqa.suse.de/tests/overview?version=15-SP2&build=b10n1k%2Fos-autoinst-distri-opensuse%238929&distri=sle)

### CODE REVIEW

**Comment by jknphy** [comment]
As discussed offline, let's create a test matrix using this [link](https://openqa.suse.de/tests/overview?version=15-SP2&flavor=Full&build=b10n1k%2Fos-autoinst-distri-opensuse%238929&distri=sle) with updated VRs.

File: schedule/yast/skip_registration/skip_registration@svirt-hyperv.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name:           skip_registration@svirt-hyperv
+description:    >
+  Like a standard scenario with explicit skipping of SCC registration
+  in case where we register by default, e.g. for SLE >= 15
+  See https://progress.opensuse.org/issues/25264 for details.
+
+schedule:
+  #- installation/bootloader_hyperv
```
Comment (OleksandrOrlov) [inline schedule/yast/skip_registration/skip_registration@svirt-hyperv.yaml]: I guess the commented line is not needed anymore. Could you please remove commented line from the commit.

Comment (b10n1k) [reply schedule/yast/skip_registration/skip_registration@svirt-hyperv.yaml]: sure thing.

**Comment by jknphy** [comment]
Please, address changes needed in the Job Group as well.

**Pull request merged**